pyquil/device/_main.py

```diff
@@ -29,6 +29,12 @@
 PERFECT_DURATION = 1 / 100
 DEFAULT_CZ_DURATION = 200
 DEFAULT_CZ_FIDELITY = 0.89
+DEFAULT_ISWAP_DURATION = 200
```

Comment:
just curious what these are for

Reply:
`quilc` makes various decisions regarding what compilation subroutine to apply or where to inject a `SWAP` instruction based on the reported fidelities and/or durations of the native gates. We do supply `quilc` with honest fidelity values, so these goofy defaults typically get overwritten, but we don't supply `quilc` with honest duration values, so these goofy defaults persist. A long time ago I asked someone to take a swag at these different numbers and this is what they told me, but repeatedly I've heard that they aren't very grounded in reality.

For what it's worth, it's mostly their relative magnitudes that matter, so nailing them isn't so important.